guides/common/modules/proc_preparing-for-using-external-databases.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Preparing_for_using_external_databases_{context}"]
+= Preparing {Project} for using external databases
+
+By default, the {Project} installation process includes installing a PostgreSQL database on the same host as {ProjectServer}.
+However, in certain {Project} deployments, using external databases instead of the default local databases can help with the server load.
```

Comment:
This is probably not nicely worded, but there are also organizations with dedicated database admins.
```suggestion
However, in certain {Project} deployments, using external databases instead of the default local databases is desirable.
For example, to help with the server load or in organizations with a specialized database team.
```